portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+ // Copyright 2019 Google LLC
```

Comment:
nit - revert this line change (looks like just an extra space)

Reply:
thanks! updated!

Reply:
Looks like there's still an extra space before the `//`